schema.graphql

```diff
@@ -698,6 +717,16 @@ type Mutation {
     input: [SaveUpdateTagsInput!]! # @constraint(minItems: 1, maxItems: 30),
     timestamp: ISOString!
   ): SaveWriteMutationPayload! @tag(name: "alpha")
+
+  """
+  Creates a new Save; if the Save already exists (either in List or Archive), "re-add" it.
+  "Re-adding" unarchives and undeletes the Save as applicable, and refreshes the "createdAt"
+  timestamp.
+  """
+  saveUpsert(
+  input: [SaveUpsertInput!]! # @constraint(minItems: 1, maxItems: 30),
+  timestamp: ISOString!
```

Comment:
calling out `timestamp` is here for all saves. we don't need to mention them individually for each SaveUpsertInput https://github.com/Pocket/list-api/pull/701/files#r1149611093